synfig-studio/src/gui/widgets/widget_waypoint.cpp

```diff
@@ -302,6 +286,7 @@ Widget_Waypoint::config_tcb_params(bool show_params)
 		adj_temporal_tension->set_value(waypoint.get_temporal_tension());
 	}
 
+	tcbFrame->set_visible(show_params);
 	tcbGrid->set_visible(show_params);
```

Comment:
if the latter is added inside to the former we only need the former to be hidden